Python/CodeChef_Selenium/codechef_selenium_script.py

```diff
@@ -0,0 +1,79 @@
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from time import sleep
+from selenium.webdriver.common.action_chains import ActionChains
+import os
+
```

Comment:
Good work @HaripriyaB only add proper comments in your code.
Rest LGTM

Reply:
Sure @yashaswibiyahut